purpleair-aqi.js

```diff
@@ -89,184 +128,191 @@ const levelAttributes = [
   },
 ];
 
-// Get level attributes for AQI
-function getLevelAttributes(level, attributes) {
-  let applicableAttributes = attributes
-    .filter((c) => level > c.threshold)
-    .sort((a, b) => b.threshold - a.threshold);
-  return applicableAttributes[0];
```

Comment:
Since `attributes` is already sorted, we don't need to sort it again here.

After removing the `.sort`, this function became so simple that I removed it altogether.